Assets/StreamingAssets/LUA/Tile.lua

```diff
@@ -6,7 +6,6 @@
 -------------------------------------------------------
 
 -------------------------------- Tile Actions --------------------------------
-
 function CanBuildHere_Standard ( tile )
 	return true
```

Comment:
We've been accepting PR's that convert tabs to 4 spaces, so we are moving in that direction.